src/core/thyme.js

```diff
@@ -79,3 +79,59 @@ export function totalProjectTime(
   return projectTimeEntries(project, time, from, to)
     .reduce((total, entry) => total + calculateDuration(entry.start, entry.end), 0);
 }
+
+
+export function pad(number: number): string {
```

Comment:
We already use `left-pad` in this file, so you don't have to implement your own version.